boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -259,38 +266,96 @@ Building
    can find the image generation tool. This binary image can be used
    to flash the SPI chip.
 
+Wiring
+========
+#. Connect the SPI Dongle ASSY 6791 to ``J44`` in the EVB.
+
+   .. image:: ./spidongle_assy6791_view1.png
+        :width: 400px
+        :align: center
+        :alt: SPI DONGLE ASSY 6791 Connected
+
+#. Connect programmer to the header J6 on the Assy6791 board, it will flash the SPI NOR chip ``U3``
+   Make sure that your programmer's offset is 0x0.
+   For programming you can use Dediprog SF100 or a similar tool for flashing SPI chips.
+
+   .. list-table:: Microchip board wiring
+      :align: center
+
+      * - .. image:: spidongle_assy6791.png
```

Comment:
OK, I've verified that this works (the ``.. image`` directive can only have white-space before it to work right):
```
      * -
          .. image:: spidongle_assy6791.png
             :width: 300px
             :align: center
             :alt: SPI DONGLE ASSY 6791

        -
          .. image:: spidongle_assy6791_view2.png
             :width: 300px
             :align: center
             :alt: SPI DONGLE ASSY 6791 view 2
```

BTW, PR #21509 will fix the doc processing you've encountered :)